docs/contributor/releases.md

```diff
@@ -0,0 +1,45 @@
+# Releases
+
+Claircore releases are cut when they are needed, as judged by the maintainers.
+
+Releases are made from the `main` branch.
+On rare occasions when a fix is time-sensitive, it is possible to create a release branch and make a release from it.
+
+## Process
+
+> **_NOTE:_** Ensure changelog entries have been created for the relevant commits.
+> (see [Changelog documentation](./changelog.md))
+
+### From main
+
+```sh
+NEW_VERSION=v0.999.999
+.github/scripts/prepare-release -b main -r upstream "$NEW_VERSION"
+```
+
+Follow the `prepare-release` command's instructions to merge changelog updates and release the tag.
+
+### From release branch
+
+First create the relevant release branch (e.g if you are releasing `v1.6.1` create `release-1.6` from the
+previous tag (in this case `v1.6.0`)). Then backport any commits and push up the release branch.
```

Comment:
```suggestion
First, create the relevant release branch.
For example, if you are releasing `v1.6.1` create `release-1.6` from the
previous tag (in this case, `v1.6.0`).

Next, cherry-pick any needed commits with the `-x` flag to keep a reference to the original commit.
This may involve rewriting the changes.

Once the backports are done, push the release branch.
```

Reply:
I'm going to update this to reflect the version/branch examples from the snippets